lightning/lightning.go

```diff
@@ -207,7 +221,7 @@ func (l *Lightning) RunServer() error {
 
 var taskCfgRecorderKey struct{}
 
-func (l *Lightning) run(taskCfg *config.Config) (err error) {
+func (l *Lightning) run(taskCfg *config.Config, glues ...glue.Glue) (err error) {
```

Comment:
Are there any use cases that need multi glues?

Reply:
> Are there any use cases that need multi glues?

No, just to make old `run(cfg)` could work

Reply:
`run` is a private function you could change all invocation to do `.run(cfg, nil)` or something

Reply:
https://github.com/pingcap/tidb-lightning/pull/456/commits/6e4da0a4257a061a0882b3de76b3cafb661f0682